src/python/grpcio_tests/tests/unit/_server_ssl_cert_config_test.py

```diff
@@ -154,107 +154,110 @@ def _do_one_shot_client_rpc(self,
         del client_stub
 
     def _do_common_test_recipe(self):
```

Comment:
This method name can be as short as `_test` (right?).

Reply:
done